### ISSUE TITLE
Jeeves and Repair Bot

### ISSUE THREAD

**Issue by GalaxyWoW** [tkrokli/TrinityCore#1]
Engineering

Jeeves has neutral faction but u can still trade with him but it must be friendly, unhitable

Repair Bot - neutral fac, can be used for repairs but also i killable, must be friendly

**Comment by tkrokli** [comment]
Issues should include the core revision hash (and TDB version, if possible). I am assuming that this issue is based on Jeutie's Blizzlike Repack 1.4, which has got the core revision hash https://github.com/tkrokli/TrinityCore/commit/3761d241196827c5d12f2af0ce1c124d9172fc82 (and the DB version TDB 335.58 + all updates up to the core revision), but I would prefer that you confirm this in the same post as the issue.

To sum up your issue, am I correct in assuming that you are saying that the repair bots have got the wrong faction? If I am able to verify this (and even claim enough evidence to open a new issue in the TrinityCore issue tracker), the faction needs verification from sniffs done on retail wow (from Blizzard servers).

**Comment by GalaxyWoW** [comment]
yes it is the Jeutie's Blizzlike Repack 1.4

**Comment by tkrokli** [comment]
OK, thanks for the confirmation. May I also ask of you to write a short instruction on how to replicate the issues, since I rarely do engineering and have earlier only summoned those bots by either **.add [NPC ID]** or **.learn [spell ID]** . (This is just so I don't waste too much time trying to find out if I need to level up everything manually in Engineering or if there are shortcuts that won't block me from replicating the issue.)

**Comment by GalaxyWoW** [comment]
.setskill 202 450 450
go find enginnering trainer and train ( can be skipped too)
.add item 49040     (Jeeves)
.add item 40769     (Scrapbot Construction Kit)
.add item 34113     (Field Repair Bot 110G)


PS: i think the issue can be fixed from the SQL table creature_template and changing faction there, but i didn't try it because i cant figure what is the right number to put there.

**Comment by tkrokli** [comment]
> PS: I think the issue can be fixed from the SQL table creature_template and changing faction there, but i didn't try it because I cant figure what is the right number to put there.

Yes, it is best not to change the faction number before you are reasonably certain that the faction number is either sourced from reliable material or from sniffs. I don't know anything about handling sniffs myself yet, since only the TrinityCore developers (and those developers they trust) handle the sniff results.

I can't promise exactly when I will have time to follow up on this issue, but at least it is here in this list and I will be able to pick it up when I find the time or when I have got any clues about if this bug is real and live in the TrinityCore source (I need to get it confirmed by some of the TrinityCore developers).

**Comment by GalaxyWoW** [comment]
the faction is not that important as long as the work properly, in this case they are, you can still trade and repair, also i know 1-2 issues too also am gona add them too when i run some tests on rev 1.4, just minor bugs, Thanks for your respond and efforts ! Have a nice day.
